ElectronicObserverCoreTests/BattleRank/BattleRankTests.cs

```diff
@@ -0,0 +1,470 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using ElectronicObserver.Data.Battle;
+using ElectronicObserver.Database;
+using ElectronicObserver.Database.Sortie;
+using ElectronicObserver.Services;
+using ElectronicObserver.Window.Tools.SortieRecordViewer;
+using ElectronicObserver.Window.Tools.SortieRecordViewer.Sortie.Node;
+using ElectronicObserver.Window.Tools.SortieRecordViewer.SortieDetail;
+using ElectronicObserverTypes;
+using ElectronicObserverTypes.Mocks;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+
+namespace ElectronicObserverCoreTests.BattleRank;
+
+[Collection(DatabaseCollection.Name)]
+public class BattleRankTests(DatabaseFixture dataBase)
```

Comment:
I generally prefer assigning the primary constructor to a property and then using the property in the code.
```cs
public class BattleRankTests(DatabaseFixture dataBase)
{
	private DatabaseFixture Database { get; } = dataBase;
...
```